airflow/providers/google/cloud/hooks/vertex_ai/batch_prediction_job.py

```diff
@@ -358,3 +514,98 @@ def list_batch_prediction_jobs(
             metadata=metadata,
         )
         return result
+
+
+class BatchPredictionJobAsyncHook(GoogleBaseHook):
+    """Hook for Google Cloud Vertex AI Batch Prediction Job Async APIs."""
+
+    def __init__(
+        self,
+        gcp_conn_id: str = "google_cloud_default",
+        impersonation_chain: str | Sequence[str] | None = None,
+        **kwargs,
+    ):
+        super().__init__(
+            gcp_conn_id=gcp_conn_id,
+            impersonation_chain=impersonation_chain,
+            **kwargs,
+        )
+
+    @lru_cache
+    def get_job_service_client(self, region: str | None = None) -> JobServiceAsyncClient:
```

Comment:
Maybe this method should be an async one? It is admittedly small, but I noticed that all methods of  `GoogleBaseAsyncHook` are asynchronous, even the most basic.

Reply:
Although it is async hook, the `__init__()` method is synchronous, thus we shouldn't change that. There are also no API calls in that method, that's why it simply doesn't need to use async approach. Moreover, when we use async hooks, we instantiate them synchronously.